go/consensus/tendermint/abci/mux.go

```diff
@@ -542,15 +542,15 @@ func (mux *abciMux) decodeTx(ctx *api.Context, rawTx []byte) (*transaction.Trans
 	return &tx, &sigTx, nil
 }
 
-func (mux *abciMux) processTx(ctx *api.Context, tx *transaction.Transaction, txSize int) error {
+func (mux *abciMux) processTx(ctx *api.Context, tx *transaction.Transaction, txSize int) (int64, error) {
```

Comment:
I think it would be better if the caller extracted the priority from the context instead (similar to how it is for gas use).

Reply:
Oh, right, I forgot to change this when I moved priority handling into the context :)